src/app/facility/analysis/run-analysis/group-analysis/regression-model-selection/regression-model-selection.component.html

```diff
@@ -115,9 +115,11 @@
                             &mdash;
                         </span>
                     </td>
-                    <td *ngIf="showInvalid && !model.errorModeling">
-                        <span *ngIf="model.isValid">Valid</span>
-                        <span *ngIf="!model.isValid">Invalid</span>
+                     <td *ngIf="showInvalid && !model.errorModeling">
+                        <span *ngIf="model.isValid">&mdash;</span>
+                        <span *ngFor="let notes of model.modelValidationNotes">
```

Comment:
Old models won't have modelValidationNotes object. Can we double check that this doesn't break older models please